packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -208,6 +209,15 @@ class FlutterDevice {
           getSkSLMethod: getSkSLMethod,
           device: device,
         );
+        await service.streamListen(EventStreams.kExtension);
```

Comment:
Where this is located it currently won't work for web builds, since the setup there is a bit more involved. It would also need to be added somewhere to resident_web_runner.dart